types.go

```diff
@@ -23,9 +23,17 @@ type (
 		URL     string
 	}
 
+	// GIF type.
+	GIF struct {
+		ID      string
+		Preview string
+		URL     string
+	}
+
 	// Tweet type.
 	Tweet struct {
 		ConversationID    string
+		GIFs              []GIF
```

Comment:
This change is agruable. I made GIFs a separate array field on the `Tweet` struct. It seems to give stronger typing. However, GIFs are basically soundless MP4 in Twitter's API. We could also treat them as a `Video`. So instead, we could add a `kind: "animated_gif/video"` field to the `Video` struct. I don't have a strong opinion on which approach is better. I chose the approach of creating a separate type and field for stronger typing